azure-pipelines.yml

```diff
@@ -39,10 +39,10 @@ pr:
 jobs:
 - job: Bedrock_Test_Harness
   displayName: Bedrock_Build_Tests
-  timeoutInMinutes: 90
+  timeoutInMinutes: 180
```

Comment:
Before merging, we need to figure out a way to parallelize our integration tests to avoid waiting 3 hours for the entire test suite.  @jmspring